src/runes/varint.rs

```diff
@@ -0,0 +1,130 @@
+use super::*;
+
+#[cfg(test)]
+pub fn encode(n: u128) -> Vec<u8> {
```

Comment:
should `encode` only be for tests?

Reply:
Currently it's only for tests, since the non-test code uses `encode_to`, which is more efficient.